doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -137,14 +152,6 @@ nRF9160 samples
 
   * Modified to use runtime location service selection instead of compile-time configurations.
 
-Matter samples
```

Comment:
Is this a side effect of resolving a rebase conflict?

Reply:
Yes, the entries should be ordered alphabetically, so I moved matter into correct place.